ckan-backend-dev/src/ckanext-wri/ckanext/wri/logic/action/get.py

```diff
@@ -317,6 +319,12 @@ def package_search(context: Context, data_dict: DataDict) -> ActionResult.Packag
                 if isinstance(package, str):
                     package = {result_fl[0]: package}
                 extras = cast("dict[str, Any]", package.pop("extras", {}))
+                
+                if return_user:
```

Comment:
Have you checked if when it returns the user its not returning their password hash also? I know ckan has that problem, so pls make sure of that, either by removing unnecessary fields or just returning the bare minimum needed